Makefile

```
@@ -3,7 +3,7 @@
 help:
 	@echo "  clean-build - remove build artifacts"
 	@echo "  clean-pyc - remove Python file artifacts"
-	@echo "  lint - check style with flake8"
+	@echo "  lint - check style with flake8, isort and black"
```

We should add a target for `format`

```suggestion
	@echo "  lint - check style with flake8, isort and black"
	@echo "  format - autoformat with autoflake, isort and black"
```